optuna/trial/_trial.py

```diff
@@ -586,21 +585,11 @@ def should_prune(self, step=None):
             Please refer to the example code in :func:`optuna.trial.Trial.report`.
 
         Args:
```

Comment:
I think we can remove `Args:` too.

![image](https://user-images.githubusercontent.com/3255979/84620995-68c26b00-af14-11ea-866f-08309c8ba25c.png)